.github/workflows/run-analysis.yml

```diff
@@ -0,0 +1,31 @@
+name: Run Analysis
+
+# Controls when the action will run.
+on:
+  workflow_dispatch:
```

Comment:
I tend to leave `workflow_dispatch` in my actions so they can be run outside of pushing a new commit. If this is an issue, please let me know and I'll remove it.